nrf_security/src/core/nrf_oberon/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ target_include_directories(mbedcrypto_common
 target_include_directories(mbedcrypto_common
   INTERFACE
     # Nordic PSA headers
-    ${NRF_SECURITY_ROOT}/include/
+    ${NRF_SECURITY_ROOT}/include
```

Comment:
No issue with this line, this and the next commits already exist in https://github.com/nrfconnect/sdk-nrfxlib/pull/1010/commits.
So better remove them from here.

Reply:
1010 is now rebased on top of 1014 so there are no duplicates in the branches (although github shows the same commits up for review in several PRs).